CHANGELOG.rst

```diff
@@ -94,3 +117,4 @@ Version `0.2.0`_
 .. _`0.4.0`: https://github.com/GreyNoise-Intelligence/pygreynoise/compare/v0.3.0...0.4.0
 .. _`0.4.1`: https://github.com/GreyNoise-Intelligence/pygreynoise/compare/v0.4.0...0.4.1
 .. _`0.5.0`: https://github.com/GreyNoise-Intelligence/pygreynoise/compare/v0.4.1...0.5.0
+.. _`dev`: https://github.com/GreyNoise-Intelligence/pygreynoise/compare/v0.5.0...HEAD
```

Comment:
I believe this was all manually deleted/updated with 0.4.1 thus causing the issues with `bumpversion` in the `0.5.0` release.